server/services/prisonerInformation.js

```diff
@@ -4,7 +4,7 @@ const { logger } = require('../utils/logger');
 
 function createPrisonFormatter(listOfPrisons, key) {
   const lookup = listOfPrisons.reduce((a, p) => {
-    a.set(p.agencyId, p.formattedDescription);
+    a.set(p.agencyId, p.description);
```

Comment:
Coding style query here really. Favour destructuring? Auto return single line? I should probably check our linting preferences too actually 🤔

`listOfPrisons.reduce((a, { agencyId, description } ) => 
    a.set(agencyId, description), newMap());`
 
...also I think you _could_ possibly make it clearer we're returning a Map (I think this is reasonably widely supported)...

`new Map(listOfPrisons.map(({ agencyId, description }) => [agencyId, description]));`

Reply:
Didn't know that set returned the map!
I like the second example too